go.mod

```diff
@@ -1,23 +1,62 @@
 module github.com/kubernetes-csi/csi-lib-utils
 
-go 1.16
+go 1.19
 
 require (
 	github.com/container-storage-interface/spec v1.5.0
 	github.com/golang/protobuf v1.5.2
-	github.com/stretchr/testify v1.7.0
-	golang.org/x/net v0.0.0-20210520170846-37e1c6afe023
-	google.golang.org/grpc v1.38.0
-	k8s.io/api v0.22.0
-	k8s.io/client-go v0.22.0
+	github.com/stretchr/testify v1.8.0
+	golang.org/x/net v0.3.1-0.20221206200815-1e63c2f08a10
+	google.golang.org/grpc v1.49.0
+	k8s.io/api v0.26.0
+	k8s.io/client-go v0.26.0
 	k8s.io/component-base v0.22.0
-	k8s.io/klog/v2 v2.9.0
+	k8s.io/klog/v2 v2.80.1
```

Comment:
k8s.io/component-base v0.22.0?

Can this be updated to 0.26.0?

Reply:
Update to 0.26.0 break the unit tests as we see above.

Reply:
I see.  Can you add a comment about this in the PR description?

Reply:
I think we should update the component-base and fix the tests. I'm not sure if there will be any side effects if the sidecars and the library use two different versions of component-base: https://github.com/kubernetes-csi/external-provisioner/blob/master/go.mod#L24